pyrogram/methods/messages/get_media_group.py

```diff
@@ -48,10 +48,15 @@ async def get_media_group(
         Raises:
             ValueError: In case the passed message id doesn't belong to a media group.
         """
-        # There can be maximum 10 items in a media group. 
-        messages = await self.get_messages(chat_id, [msg_id for msg_id in range(message_id - 9, message_id + 10)], replies=0)
+        messages = await self.get_messages(
+            chat_id=chat_id,
+            message_ids=[msg_id for msg_id in range(message_id - 9, message_id + 10)],
+            replies=0
+        )
 
-        media_group_id = messages[9].media_group_id
+        # There can be maximum 10 items in a media group.
+        # The if/else condition to fix the problem of getting correct `media_group_id` when it has `message_id` less then 10.
+        media_group_id = messages[9].media_group_id if len(messages) == 19 else messages[message_id-1].media_group_id
```

Comment:
Maybe we should search again for the message having the same id we passed to the method instead of relying on indexes.

Reply:
That's true, but when we first created this method with Legenda24, he was very concious about the speed.

And by adding those conditions we still have the same speed and 100% method. (I really doubt Client API will ever return `Messages` with negative `message_id`s).

Reply:
Anyway this case is very very rare. Never seen that somebody faces with it. (in main chat)

Reply:
Well, but I personally faced this. 

And including the fact that's it's 'our' method, I want it to be as good as possible :D

Reply:
at the moment it's good as possible

Reply:
Well, if you mean this with this PR it will become as good as possible, then I totally agree.

Reply:
> I want it to be as good as possible :D
>> its good as possible

Reply:
>  if you mean that with this PR it will become as good as possible, then I totally agree.

Just like I said.

Reply:
I said it like 2-3 replies earlier. don't need to explicit what I meant